app/scripts/components/data-catalog/container.tsx

```diff
@@ -15,15 +18,32 @@ import { veda_faux_module_datasets } from '$data-layer/datasets';
  * veda2 instances can just use the direct component, 'DataCatalog', and manage data directly in their page views
  */
 
+const InteractiveLink = styled.a`
```

Comment:
I see no `CardInteractiveElement` is passed from Next instance from [this pr from Next instance](https://github.com/developmentseed/next-veda-ui/pull/3/files#diff-6578b281e50705562cdf8ba73c73ae2bb1df366e18e91e9595106b09cfbf3f89). Based on [this line](https://github.com/NASA-IMPACT/veda-ui/pull/1096/files#diff-c618b2febe87cadd3d3931c1498d8b40c67f5793d0235e3777dd79a202183f34R78), it will be subbed as `a` tag. But `InteractiveLink` here is more than an `a` tag. It is a styled component with a style to expand its target area. I can't seem to click on the card on this page : https://deploy-preview-3--veda-ui-next-test.netlify.app/datasets .

Also, I think the interactive link could be rendered with either `a` tag or `Link` because styled component accepts `as` as a polymorphic prop to decide which component to render (https://styled-components.com/docs/api#as-polymorphic-prop)  - if you look at the rendered html from next instance, you can see that `a` tag as a pure html doesn't know what to do with `as`.
<img width="944" alt="Screenshot 2024-08-28 at 3 31 03 PM" src="https://github.com/user-attachments/assets/aa483a7c-84e6-493a-92a5-55afe6e346af">


This is related to why I opened #1121 . We can leave the styled component as it is but pass the component only in charge of routing as a prop. 

What do you think?

Reply:
addressed